users/class_tests/logout_tests.py

```diff
@@ -1,20 +1,21 @@
 import pytest
 from users.models import User
+from .constants import VALID_USERNAME2, HOMEPAGE_URL
+from .constants import VALID_USERNAME, VALID_EMAIL, VALID_PASSWORD
```

Comment:
instead of repeating the "from .constants" 2 times, you can do it like:
```suggestion
from .constants import (
    VALID_USERNAME2, HOMEPAGE_URL, VALID_USERNAME, VALID_EMAIL, VALID_PASSWORD
)
```

Reply:
done